src/module-vsbridge-indexer-agreement/Index/Mapping/Agreement.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright Divante Sp. z o.o.
+ * See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Divante\VsbridgeIndexerAgreement\Index\Mapping;
+
+use Divante\VsbridgeIndexerCore\Api\Mapping\FieldInterface;
+use Divante\VsbridgeIndexerCore\Api\MappingInterface;
+use Magento\Framework\DataObject;
+
+class Agreement implements MappingInterface
+{
+    /**
+     * @return array|mixed|null
+     */
+    public function getMappingProperties()
+    {
+        $properties = [
+            'id' => ['type' => FieldInterface::TYPE_LONG],
+            'content' => ['type' => FieldInterface::TYPE_TEXT],
+            'active' => ['type' => FieldInterface::TYPE_BOOLEAN],
+            'name' => ['type' => FieldInterface::TYPE_TEXT],
+            'content_height' => ['type' => FieldInterface::TYPE_TEXT],
+            'checkbox_text' => ['type' => FieldInterface::TYPE_TEXT],
+            'is_html' => ['type' => FieldInterface::TYPE_BOOLEAN],
+            'mode' => ['type' => FieldInterface::TYPE_INTEGER],
+        ];
+
+        $mappingObject = new DataObject();
+        $mappingObject->setData('properties', $properties);
+
+        return $mappingObject->getData();
```

Comment:
we really don't need this, you can just do `return $properties`

Reply:
fixed